algebras/algebra/src/main/scala/endpoints/algebra/Requests.scala

```diff
@@ -89,5 +89,27 @@ trait Requests extends Urls with Methods with InvariantFunctorSyntax with Semigr
     headers: RequestHeaders[HeadersP] = emptyHeaders
   )(implicit tuplerUB: Tupler.Aux[UrlP, BodyP, UrlAndBodyPTupled], tuplerUBH: Tupler.Aux[UrlAndBodyPTupled, HeadersP, Out]): Request[Out] = request(Post, url, entity, headers)
 
+  /**
+    * Helper method to perform UPDATE request
+    * @tparam UrlP Payload carried by url
+    * @tparam BodyP Payload carried by body
+    * @tparam HeadersP Payload carried by headers
+    * @tparam UrlAndBodyPTupled Payloads of Url and Body tupled together by [[Tupler]]
+    */
+  final def update[UrlP, BodyP, HeadersP, UrlAndBodyPTupled, Out](
```

Comment:
Actually, since the name of the HTTP verb is “PUT” we should name this method `put` as well, instead of `update`.